src/main/java/Cart.java

```diff
@@ -0,0 +1,11 @@
+import java.util.List;
+
+public class Cart {
+    public final String itemName;
+    public final Integer qty;
```

Comment:
If generic type is not needed, please avoid to use it.
Integer -> int

https://softwareengineering.stackexchange.com/questions/203970/when-to-use-primitive-vs-class-in-java

Reply:
Updated as advised